keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -93,40 +93,52 @@ to begin with? AdminNetworkPolicy is a popular alternative.
 
 ## Motivation
 
-Kubernetes provides NetworkPolicy resources to control traffic within a
-cluster. NetworkPolicies focus on expressing a developers intent to secure
-their applications. Thus, in order to satisfy the needs of a security admin,
-we propose to introduce a new API that captures the administrators intent.
+Kubernetes provides the NetworkPolicy resource to control traffic within a
+cluster. NetworkPolicy focuses on expressing a developer's intent to secure
+their applications. However, it was not intended to be used for administrative
+traffic control, which is reflected by its design:
+- It has no concept of explicit "deny" rules, because the application deployer can
+  simply refrain from allowing the things they want to deny.
+- It doesn't include priorities, or any concept of one user "outranking" another
+  and being able to override their policies, because a given application is expected
+  to only have policies created by a single user (the one deploying the application).
+Thus, in order to satisfy the needs of a security admin, we propose to introduce
+a new API that captures the administrator's intent.
 
 ### Goals
 
 The goals for this KEP are to satisfy the following key user stories:
 1. As a cluster administrator, I want to enforce irrevocable guardrails that
    all workloads must adhere to in order to guarantee the safety of my clusters.
    In particular I want to enforce certain network level access controls that are
-cluster scoped and cannot be overridden or bypassed by namespace scoped network policies.
-2. As a cluster administrator, I want to have the option to enforce network level access controls
-   that facilitate network multi-tenancy and strict network level isolation between multiple
-   teams and tenants sharing a cluster via use of namespaces or groupings of namespaces per tenant.
-3. As a cluster administrator, I want to optionally also deploy an additional default set of policies to all
-   workloads that may be overridden by the developers if needed.
+   cluster scoped and cannot be overridden or bypassed by namespace-scoped network
+   policies. For example, I want to have the option to enforce network level access
+   controls that facilitate network multi-tenancy and strict network level isolation
+   between multiple teams and tenants sharing a cluster via use of namespaces or
+   groupings of namespaces per tenant.
+2. As a cluster administrator, I want to optionally also deploy an additional default
+   set of policies to all workloads that may be overridden by the developers if
+   needed.
```

Comment:
I had taken a stab at this by providing some concrete examples, and I'm not sure we should necessarily remove 3. here 

```
The goals for this KEP are to satisfy the following key user stories:

1. As a cluster administrator, I want to enforce irrevocable guardrails that
   all workloads must adhere to in order to guarantee the safety of my clusters.
   In particular I want to enforce certain network level access controls that are
   cluster scoped and cannot be overridden or bypassed by namespace scoped network policies.

   Example: I would like to explicitly allow all pods in my cluster to reach 
   kubeDNS. 

2. As a cluster administrator, I want to have the option to enforce network level access controls
   that facilitate network multi-tenancy and strict network level isolation between multiple
   teams and tenants sharing a cluster via use of namespaces or groupings of namespaces per tenant.
   
   Example: I would like to define two tenants in my cluster, one composed of the pods 
   in `foo-ns-1` and `foo-ns-2` and the with the pods in `bar-ns-1`, where inter-tenant traffic is 
   denied. 

3. As a cluster administrator, I want to optionally also deploy an additional default set of policies 
   to all workloads that may be overridden by the developers if needed 

   Example: I would like to explicitly delegate the restriction of traffic destined 
   for cluster monitoring pods to the developer, allowing them to setup network policy to 
   deny or allow the traffic from/to their application. 
```